pkg/devserver/devserver.go

```diff
@@ -166,6 +171,7 @@ func start(ctx context.Context, opts StartOpts) error {
 		executor.WithState(sm),
 		executor.WithServiceQueue(queue),
 		executor.WithServiceExecutor(exec),
+		executor.WithServiceDebouncer(debouncer),
```

Comment:
why is `debouncer` needed for both the executor instance and the executor service?